src/oc_erchef/apps/oc_chef_wm/test/oc_chef_action_tests.erl

```diff
@@ -301,7 +301,8 @@ end_to_end_test_() ->
     State = #base_state{requestor = #chef_requestor{name = <<"rob">>, type = <<"user">>},
                         reqid = <<"Xfh5mCQvjRgWDdlevrdyGt8M4lecXmN3gpGXrKKiUYqKdeD3">>,
                         organization_name = <<"cmwest">>,
-                        resource_state=#node_state{node_data = {[{<<"name">>,<<"db">>}]} }},
+                        resource_state=#node_state{node_data = {[{<<"name">>,<<"db">>},
+                                                                 {<<"password">>,<<"changeme">>}]} }},
```

Comment:
You should be able to test just the `maybe_scrub_password` function to make sure that it doesn't add it.